Package.swift

```diff
@@ -49,6 +49,10 @@ let package = Package(
         .product(name: "FlatBuffers", package: "flatbuffers"),
       ]
     ),
+    .executableTarget(
+      name: "SQLRepl",
```

Comment:
Do we need to add `Spark` prefix? I.e., `SparkSQLRepl`?

Reply:
Thank you. Sure, let me add it. :)